uint8 SubsystemHeader    # Byte0
uint64 SubsystemData     # Byte1..5
uint8 UserConfig         # Byte6
uint8 MessageIdentifier  # Byte7
